apps/web/pages/event-types/[type].tsx

```diff
@@ -22,6 +22,7 @@ import timezone from "dayjs/plugin/timezone";
 import utc from "dayjs/plugin/utc";
 import { GetServerSidePropsContext } from "next";
 import { useRouter } from "next/router";
+import form from "pages/sandbox/form";
```

Comment:
Are we supposed to be using things from the sandbox?

Reply:
Good Spot - Its not even being used... Will remove this

Reply:
New commit added